src/vaadin-date-picker-mixin.html

```diff
@@ -699,11 +699,11 @@
      * @param {string} value Value to validate. Optional, defaults to user's input value.
      * @return {boolean} True if the value is valid and sets the `invalid` flag appropriately
      */
-    validate(value) {
-      // reset invalid state on the underlying text field
-      this.invalid = false;
-      value = value !== undefined ? value : this._inputValue;
-      return !(this.invalid = !this.checkValidity(value));
+    validate() {
```

Comment:
Wouldn't removing the `value` argument make this a breaking change?

Reply:
I guess it should not, can be considered as fix/cleanup. 
Because the validation will return the same result as previously based on the `checkValidity` `inputValid` which was checking the equality with the current `_selectedDate `. The only case when it could be different is when you manually providing the value that equals `_selectedDate` to pass the validation, but that should be achieved with custom validator / disabled validation. 🤔

Reply:
I agree that this fixes previously incorrectly working behavior.

Let's remember to mention that as "Behavior altering change" in the release notes.